doc/appendix-de-kassensichv/examples/pos-receipt.md

```diff
@@ -0,0 +1,468 @@
+    }
+  ],
+  // 0x4445 0000 0000 0001 (pos-receipt) 
+  "ftReceiptCase": 4919338167972134913,
+  "ftReceiptCaseData": "",
+  "cbReceiptAmount": 13.00,
+  "cbUser": "Chef",
+  "cbArea": "Tisch 01",
+  "cbSettlement": "Test1"
+}
+`
+
+
+ReceiptResponse
+
+`
+{
+    "ftCashBoxID": "cashboxid-guid",
+    "ftQueueID": "queueid-guid",
+    "ftQueueItemID": "queueitemid-guid",
+    "ftQueueRow": 96,
+    "cbTerminalID": "1",
+    "cbReceiptReference": "1",
+    "ftCashBoxIdentification": "[ftCashBoxIdentification]",
+    "ftReceiptIdentification": "ft[ftQueueRow]#T[finishTransactionNr]",
+    "ftReceiptMoment": "2019-10-25T13:32:46.133Z",
+    "ftSignatures": [
+        {
+            "ftSignatureFormat": 3,
+            //0x4445000000000001 (qr-code according kasssichv)
+            "ftSignatureType": 4919338167972134913,
+            "caption": "www.fiskaltrust.de",
+            "data": "V0;[ftCashBoxIdentification];Kassenbeleg-V1;Beleg^3.80_9.20_0.00_0.00_0.00^13.00:Bar;[finishTransactionNr];[signatureCounter];[startTransactionMoment];[finishTransactionMoment];[signatureAlgorithm];[logTimeFormat e.g. unixtime];[signature];[publickey]"
+        },
+        {
+            "ftSignatureFormat":13,
+            //0x4445000000000010 (start-transaction-result)
+            "ftSignatureType": 4919338167972134928,
+            "caption": "start-transaction-signature",
+            "data": "[startTransactionResult]"
+        },
+        {
+            "ftSignatureFormat":13,
+            //0x4445000000000011 (finish-transaction-payload)
+            "ftSignatureType": 4919338167972134929,
+            "caption": "finish-transaction-payload",
+            "data": "[finishTransactionPayload]"
+        },
+        {
+            "ftSignatureFormat": 13,
+            //0x4445000000000012 (finish-transaction-result)
+            "ftSignatureType": 4919338167972134930,
+            "caption": "finish-transaction-payload",
+            "data": "[finishTransactionResult]"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x4445000000000013 (qr-code-version)
+            "ftSignatureType": 4919338167972134931,
+            "caption": "qr-code-version",
+            "data": "V0"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x4445000000000014 (processType)
+            "ftSignatureType": 4919338167972134933,
+            "caption": "processType",
+            "data": "Kassenbeleg-V1"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x4445000000000015 (processData)
+            "ftSignatureType": 4919338167972134934,
+            "caption": "processData",
+            "data": "Beleg^3.80_9.20_0.00_0.00_0.00^13.00:Bar"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x4445000000000017 (<transaktions-zähler>)
+            "ftSignatureType": 4919338167972134935,
+            "caption": "transaktions-nummer",
+            "data": "[finish-transaktions-zähler]"
```

Comment:
Are the surrounding `[` `]` on purpose ?

Reply:
mentioned to be a variable, final data will be a pure number without `[` `]`